sw/device/lib/crypto/drivers/kmac.c

```diff
@@ -136,8 +471,7 @@ kmac_error_t kmac_update(const uint8_t *data, size_t data_len, uint8_t *digest,
     if (err != kKmacOk) {
       return err;
     }
-    abs_mmio_write32(TOP_EARLGREY_KMAC_BASE_ADDR + KMAC_MSG_FIFO_REG_OFFSET,
-                     *word_ptr++);
+    abs_mmio_write32(kKmacBaseAddr + KMAC_MSG_FIFO_REG_OFFSET, *word_ptr++);
```

Comment:
nit: indexing with `i` seems clearer to me here.
```suggestion
    abs_mmio_write32(kKmacBaseAddr + KMAC_MSG_FIFO_REG_OFFSET, word_ptr[i]);
```